### DISCUSSION TITLE
Output full file path

### DISCUSSION THREAD

**Discussion by hhalaby** [ionelmc/python-hunter#113] [Q&A] [answered]
How can I tell hunter to output the full file path. All I get is [...] with the ending only even for short lines that don't exceed `repr_limit`.

A reference to the docs can also be helpful as I couldn't find something there.

Example

```
[...]ne/xxx/xxx/performance.py:113   call                  => memory_rss(pid=xxx)
[...]ne/xxx/xxx/performance.py:117   return                <= memory_rss: xxx
[...]ne/xxx/xxx/performance.py:174   return             <= measure_mem: xxx
```

**Comment by ionelmc** [comment, marked as answer]
It's filename_alignment, see
https://python-hunter.readthedocs.io/en/latest/reference.html#actions

I realise now it's not a very good name, hmmmm...

**Comment by hhalaby** [comment]
@ionelmc Thanks for the answer.
As a completely new user of this (awesome) library, I think the most un-obvious thing to learn is that there are some options to configure in the trace object and some other are in the callprinter method.
